SPECS/perl/perl.spec

```diff
@@ -9,13 +9,14 @@
 Summary:        Practical Extraction and Report Language
 Name:           perl
 Version:        5.30.3
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
Can we please modify the PR title as well, to be something more changelog friendly.